WcaOnRails/app/controllers/static_pages_controller.rb

```diff
@@ -13,8 +13,12 @@ class StaticPagesController < ApplicationController
   def home
   end
 
-  def delegates
```

Comment:
You are replacing this `delegates` method by `delegates_data`.
But I think you also need an (at least empty!) `def delegates` method to redirect the controller to the Rails template that injects the React component(?)

Reply:
I think no, because it is automatically taking <controller_name>/<method_name>.html.erb (here `static_pages/delegates.html.erb`). I have used empty functions in `panel_controller` in past as I had to restrict the access to that path based on permissions.

Reply:
Cool I never knew that Rails was smart enough to route through these defaults. Thanks for letting me know!